mbus-api/src/message_bus/v0.rs

```diff
@@ -168,6 +202,113 @@ pub trait MessageBusTrait: Sized {
         let _ = request.request().await?;
         Ok(())
     }
+
+    /// Get nexuses with filter
+    #[tracing::instrument(level = "debug", err)]
+    async fn get_nexuses(filter: Filter) -> BusResult<Vec<Nexus>> {
+        let nexuses = GetNexuses {
+            filter,
+        }
+        .request()
+        .await?;
+        Ok(nexuses.0)
+    }
+
+    /// Get nexus with filter
+    #[tracing::instrument(level = "debug", err)]
+    async fn get_nexus(filter: Filter) -> BusResult<Nexus> {
+        let nexuses = Self::get_nexuses(filter).await?;
+        only_one!(nexuses)
+    }
+
+    /// create nexus
+    #[tracing::instrument(level = "debug", err)]
+    async fn create_nexus(request: CreateNexus) -> BusResult<Nexus> {
+        let nexus = request.request().await?;
+        Ok(nexus)
+    }
+
+    /// destroy nexus
+    #[tracing::instrument(level = "debug", err)]
+    async fn destroy_nexus(request: DestroyNexus) -> BusResult<()> {
```

Comment:
For all cases like this, Clippy will start to complain (and so will our build system when upgraded) where return units are used. Also when dealing with multiple requests I would argue that having "what is destroyed" be part of the return message might be useful? So what if we would return Ok(request.xxx) ?

Reply:
Ah, yeah then we can use that way and return the UUID of the object being destroyed for example.